custom_config_data/besu.json

```
@@ -13,8 +13,14 @@
     "londonBlock": 0,
     "preMergeForkBlock": 0,
```

Think this could be changed to match the geth genesis.json (not that it's used AFAIK)

```suggestion
    "mergeNetsplitBlock": 0,
```